dm/config/subtask.go

```diff
@@ -131,9 +131,9 @@ type SubTaskConfig struct {
 	flagSet *flag.FlagSet
 
 	// when in sharding, multi dm-workers do one task
-	IsSharding      bool   `toml:"is-sharding" json:"is-sharding"`
-	ShardMode       string `toml:"shard-mode" json:"shard-mode"`
-	OnlineDDLScheme string `toml:"online-ddl-scheme" json:"online-ddl-scheme"`
```

Comment:
this breaks backward compatibility

Reply:
fix in https://github.com/pingcap/dm/pull/1904/commits/79748bbac20499065c91414137c90495a0e559b0. PTAL